CMakeLists.txt

```diff
@@ -174,12 +174,20 @@ if(UNIX)
         set(CPACK_PACKAGE_ICON "${CMAKE_CURRENT_SOURCE_DIR}/cockatrice/resources/appicon.icns")
     else()
         # linux
-        set(CPACK_GENERATOR DEB ${CPACK_GENERATOR})
+        IF(CPACK_GENERATOR STREQUAL "RPM")
+            set(CPACK_GENERATOR ${PACKAGE_GENERATOR} ${CPACK_GENERATOR})
+            set(CPACK_RPM_PACKAGE_LICENSE "GPLv2")
+            set(CPACK_RPM_PACKAGE_REQUIRES "protobuf, qt5-qttools, qt5-qtsvg, qt5-qtmultimedia")
+            set(CPACK_RPM_PACKAGE_GROUP "Amusements/Games")
+            set(CPACK_RPM_PACKAGE_URL "http://github.com/Cockatrice/Cockatrice")
+        ELSE()
+            set(CPACK_GENERATOR DEB ${CPACK_GENERATOR})
```

Comment:
Here you are prepending "DEB" to the list of generators used by cpack; i think you can just `set(CPACK_GENERATOR DEB)` since we currently doesn't support multiple generators anyway